src/Saga/SagaWorkflow.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+namespace TemporalioSamples.Saga;
+
+[Workflow]
+public class SagaWorkflow
+{
+    [WorkflowRun]
+    public async Task RunAsync(TransferDetails transfer)
+    {
+        List<Func<Task>> compensations = new();
+        var logger = Workflow.Logger;
+
+        var options = new ActivityOptions()
+        {
+            StartToCloseTimeout = TimeSpan.FromSeconds(90), // schedule a retry if the Activity function doesn't return within 90 seconds
+            RetryPolicy = new()
+            {
+                InitialInterval = TimeSpan.FromSeconds(15), // first try will occur after 15 seconds
+                BackoffCoefficient = 1, // double the delay after each retry
+                MaximumInterval = TimeSpan.FromMinutes(1), // up to a maximum delay of 1 minute
+                MaximumAttempts = 2, // fail the Activitiesivity after 2 attempts
+            },
+        };
```

Comment:
We usually don't want to encourage people to set all of these. The defaults are ideal, especially for samples. If there are specific ones for compensation, so be it, but I'd rather this be:

```suggestion
        var options = new ActivityOptions() { StartToCloseTimeout = TimeSpan.FromSeconds(90) };
```